langchain/document_transformers/doctran_text_extract.py

```diff
@@ -53,12 +53,15 @@ def __init__(
         self,
         properties: List[dict],
         openai_api_key: Optional[str] = None,
+        openai_api_model: Optional[str] = None,
     ) -> None:
         self.properties = properties
-        self.openai_api_key = openai_api_key or get_from_env(
+        self.openai_api_model = openai_api_key or get_from_env(
```

Comment:
guessing this line wasn't supposed to change?

Reply:
Definitely my mistake 😅. I've pushed up he fix.